dataland-frontend/public/index.html

```diff
@@ -12,7 +12,6 @@
             <strong>We're sorry but Dataland frontend service doesn't work properly without JavaScript enabled.
                 Please enable it to continue.</strong>
         </noscript>
-        <p><%= BASE_URL %></p>
```

Comment:
Zum Frontend:
* Die Toasts beim Hochladen von Datensätzen oder Companies verschwinden nicht - und ich kann sie auch nicht schließen.

Reply:
Ergänzung:
- Wenn ich in der Dataland-Company-Search nichts ins Suchfeld eingebe und auf Suche klicke, sollten ja alle companys erscheinen.  Der Backend-Endpunkt verhält sich jedenfalls so (leerer String = alle companys anzeigen).   Jedoch kommt eine Fehlermeldung.

Reply:
![image](https://user-images.githubusercontent.com/47553592/159736911-58f8a0a2-9dbb-458f-b4cd-c3d4093f1d52.png)

Reply:
ok